backends/smt2/smtio.py

```diff
@@ -445,6 +468,10 @@ def info(self, stmt):
 
         fields = stmt.split()
 
+        if fields[1] == "yosys-smt2-timeout":
+            self.timeout = int(fields[2])
+            assert self.timeout > 0
```

Comment:
I'm not convinced that passing a timeout through the SMT file is a sensible thing to do. We don't do that for other similar parameters, such as which SMT solver to use.

Reply:
Fair enough.  It already works fine without it, so it's superfluous, anyway.